iscsidev/iscsi.go

```diff
@@ -209,12 +207,12 @@ func LogoutTarget(target string) error {
 		 * 21"(no record found) as valid result
 		 */
 		for i := 0; i < RetryCounts; i++ {
-			if !iscsi.IsTargetDiscovered(ip, target, ne) {
+			if !iscsi.IsTargetDiscovered("", target, ne) {
 				err = nil
 				break
 			}
 
-			err = iscsi.DeleteDiscoveredTarget(ip, target, ne)
+			err = iscsi.DeleteDiscoveredTarget("", target, ne)
```

Comment:
Is this going to logout all the existing sessions for the target? If there is more than one, just in case.

Reply:
Yes. The test mentioned in the original issue is passed, which means this updated function can clean up all sessions for the target.